### PR TITLE
Oleg Betsun lasagna master exercise

### PR DESCRIPTION
**Pull request by olegbet**
Oleg Betsun lasagna master exercise

### CODE REVIEW

File: olegbetsun/lasagna_master.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+func Quantities(layers []string) (int,float64) {
+	var noodles, sauces int
+	var sauce float64
+
+	for _,v := range layers{
+		if v == "noodles" {
+			noodles++
+		} else if v == "sauce" {
+			sauces++ 
+		}
```
Comment (davidmogar) [inline olegbetsun/lasagna_master.go:22]: Better to use a switch block IMO.

File: olegbetsun/lasagna_master.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+	return len(layers)*timePerLayer
+
+}
+
+func Quantities(layers []string) (int,float64) {
+	var noodles, sauces int
+	var sauce float64
+
+	for _,v := range layers{
```
Comment (davidmogar) [inline olegbetsun/lasagna_master.go:17]: I would give it an actual name instead of just `v`.

File: olegbetsun/lasagna_master.go [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+			sauces++ 
+		}
+	}
+	noodles = noodles*50
+	sauce = float64(sauces)*0.2
+	return noodles, sauce
+}
+
+func AddSecretIngredient(friendsList []string, myList []string) []string {
+	var res []string = myList[:]
```
Comment (davidmogar) [inline olegbetsun/lasagna_master.go:30]: You can use `myList` directly. No need to create another variable.

File: olegbetsun/lasagna_master.go [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+	res = append(res, friendsList[len(friendsList)-1])
+	return res
+}
+
+
+func ScaleRecipe(amounts []float64, portions int) []float64 {
+	res := make([]float64,0,len(amounts))
+	for _,v  := range amounts{
+		res = append(res, v/2*float64(portions))
```
Comment (davidmogar) [inline olegbetsun/lasagna_master.go:39]: I would extract that operation outside the loop so you don't calculate it with each iteration.

**Review by davidmogar — COMMENTED** [review]
You should run the formatter before pushing the code. It will get rid of the extra indentation.

File: olegbetsun/lasagna_master.go (hunk 4 above)
Comment (olegbet) [reply olegbetsun/lasagna_master.go:39]: Thanks, David!

**Pull request merged**